.github/workflows/scripts/compare_translations.py

```
@@ -0,0 +1,152 @@
+    # match any in default_translation
+    for key in other_translation:
+        if key not in default_translation:
+            error_msg = f"""\
+Error Key: '{key}' - This key in '{other_file}' \
+does not match any key in '{default_file}'."""
+            errors.append(error_msg)
+
+    return errors
+
+
+def load_translation(filepath):
```

Ruff (0.11.9)

Missing return type annotation for public function `load_translation`

(ANN201)